### PR TITLE
feat: hasta la vista idle and yearn gen

### PR DESCRIPTION
**Pull request by gomesalexandre**
### Description

Follows-up on https://github.com/shapeshift/web/pull/7473 and removes unused/useless gen services:

- Yearn (has been commented out for just about forever)
- Idle (we don't support Idle as a full-fledged DeFi opportunity anymore, the only reason why we had assets data was for legacy support of users with Idle position - Portals now handles this for us)

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

> What protocols, transaction types or contract interactions might be affected by this PR?

Low to none

## Testing

- Idle assets still look sane
- Idle read-only positions (assuming you have one/many), still look sane
- As usual for all things assets gen, do a smoke test of common assets, but also *pools* assets

### Engineering

- ^
- Test against a local full assets rebuild (assets regen, color map regen not required, full cache clear), *not* against upstream develop

### Operations

- ^

## Screenshots (if applicable)

Develop (right) vs. this diff:

### Idle

Screenshot 2024-08-05 at 12 30 53
Screenshot 2024-08-05 at 12 31 14
Screenshot 2024-08-05 at 12 31 14

### Related Assets - Smoke test

https://jam.dev/c/620fc05b-9a45-4705-b915-5f0a27f2eec5

### Pool Assets - Regression test

image
image
image

### CODE REVIEW

**Comment by 0xApotheosis** [comment]
@gomesalexandre should this be targeting `release`?

**Comment by gomesalexandre** [comment]
> @gomesalexandre should this be targeting `release`?

Same for this guy, absolutely not! Aviator CLI autorug

File: src/state/migrations/index.ts [hunk 1]
```diff
@@ -93,4 +93,7 @@ export const migrations = {
   83: clearAssets,
   84: clearAssets,
   85: clearAssets,
+  86: clearAssets,
+  87: clearAssets,
+  88: clearAssets,
```
Comment (gomesalexandre) [inline src/state/migrations/index.ts:98]: rm

**Review by NeOMakinG — COMMENTED** [review]
Conceptual review looks nice, nothing so far, testing at runtime now

**Review by NeOMakinG — APPROVED** [review]
Develop
image

This branch
image


Develop
image

This branch
image

Develop
image

This branch
image

## Related assets
image
image

## Pools
image

image
image


Seems sane so far ✅

**Pull request merged**